## Hi there üëã

Welcome to the Blaser Lab Github page.  

Our lab is interested in hematopoiesis and leukemogenesis.  We also have an interest in single cell technologies. This github page is a good place to start if you want to review the code behind the figures for our papers.  From here you can also find the code for our blaseRtools R package.  This is a set of functions we use frequently in the lab related to single cell analysis, image analysis, DNA sequence manipulation, and Chip/ATAC-seq visualization.  

### [R Universe blaserlab repo!](https://blaserlab.r-universe.dev/ui#builds)

This is like a CRAN repo for everybody.  This is your best option for installing blaseRtools and blaseRtemplates.  MAC and Windows binaries are available.  See the top of the page for instructions on how to enable blaserlab repo.

### Analysis projects for papers

Here you can find links to the public github repos for our data analysis projects.  Private repositories are also available to collaborators in this area.  Most are designed to work in concert with a precompiled data package which is publicly distributed through our dryad and privately available on our OSU network drive.  

* [flt3_aml_bakerlab](https://github.com/blaserlab/flt3_aml_bakerlab)
	* Sc-RNAseq Analysis of bone marrow samples from Flt3-AML patients treated with Gilteritinib.
	* We find chemokines upregulated in patients resistant to Gilteritinib.
	* [PMID 35797240](https://pubmed.ncbi.nlm.nih.gov/35797240/)
* [hanel_lapo](https://github.com/blaserlab/hanel_lapo)
	* Sc-RNAseq Analysis of mantle cell lymphoma patient samples
	* [PMID 35831896](https://pubmed.ncbi.nlm.nih.gov/35831896/)


### [blaseRtools](https://blaserlab.github.io/blaseRtools/)

The underlying philosophy of blaseRtools is that we try to reduce the cognitive burden related to working with diverse sets of bioinformatics tools.  We try as rapidly as possible to process and funnel raw data into common data structures that are as simple to understand and work with as possible.  

The main page linked from the section title provides an overview of different "modules" of tools we have available.  From there you can link to tutorials showing common use-cases.  The tutorials are identical in content to the vignettes available in R when you install the package.

### [blaseRtemplates](https://github.com/blaserlab/blaseRtemplates)

This started out as project setup templates to be used with the usethis package.

It expanded to become general utility package for setting up projects and handling packages.

Takes some things from renv and makes them faster and more useful.

### [Datascience Workshop 2023](https://blaserlab.github.io/datascience.curriculum/)

In 2019 I began offering a small-group workshop on computational data analysis in R.  This was put on hiatus due to the COVID pandemic.  

I am happy to announce that I am once again offering the workshop in Spring 2023.

* Intended audience:  The workshop will be targeted at laboratory-based scientists (trainees and faculty) who wish to improve their ability to rigorously analyze standard laboratory data and who wish to gain the skills necessary to analyze highly-dimensional, complex datasets like scRNA-seq data.  The focus will be on practical skills for organizing and managing data, making your work rigorous and reproducible, and learning how to let the computer do the work for you.  We will not be discussing statistical theory; rather we will learn how to implement established biostatistical tools to generate results for manuscripts.  You should have a working knowledge of basic biostatistics but do not need to have prior experience with R.
* Format:  The course will be conducted virtually via zoom.  The lecture materials will be provided in advance.  You will be encouraged to read these materials, try running the code, and to prepare any questions you might have in advance.  I will go through the materials during the lecture and we will have a question and answer period at the end.  There are no tests or quizzes but you will be asked to fill out an evaluation at the end of the course.  Because of the change in format, we won‚Äôt be able to troubleshoot individual issues or do interactive coding in the live session but you will be free to work in parallel with me during the lecture.
* Syllabus:  under revision....
* Materials:  Workshop materials will be distributed via the course website.  These will be released at a later date.
* How to participate:  Email bradley.blaser@osumc.edu and be sure to include ‚Äúdatascience workshop 2023‚Äù in the subject
* Video recordings will be made available to participants.





<!--
**blaserlab/blaserlab** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
